applications/plugins/SofaPython/Binding_BaseObject.cpp

```diff
@@ -149,6 +153,7 @@ SP_CLASS_METHOD(BaseObject,getPathName)
 SP_CLASS_METHOD(BaseObject,getLinkPath)
 SP_CLASS_METHOD(BaseObject,getSlaves)
 SP_CLASS_METHOD(BaseObject,getName)
+SP_CLASS_METHOD(BaseObject,getAsACreateObjectParameter)
```

Comment:
Simply move `Binding_BaseObject.cpp`'s implementation of both `getPathName` and `getLinkPath` in `Base.cpp` as all that is used is defined in `Base` in the c++ side. (and yes I am pretty sure :-)